docker/Dockerfile.ci_cpu

```diff
@@ -109,3 +109,13 @@ RUN bash /install/ubuntu_install_androidsdk.sh
 ENV ANDROID_HOME=/opt/android-sdk-linux/
 ENV ANDROID_NDK_HOME=/opt/android-sdk-linux/ndk/21.3.6528147/
 
+# Arm(R) Ethos(TM)-U NPU driver
+COPY install/ubuntu_install_ethosu_driver_stack.sh /install/ubuntu_install_ethosu_driver_stack.sh
+RUN bash /install/ubuntu_install_ethosu_driver_stack.sh
+
+# Install Vela compiler
+COPY install/ubuntu_install_vela.sh /install/ubuntu_install_vela.sh
+RUN bash /install/ubuntu_install_vela.sh
+
+# Update PATH
+ENV PATH /opt/arm/bin:/opt/arm/cmake/bin:/opt/arm/gcc-arm-none-eabi/bin:$PATH
```

Comment:
We want to keep cmake pinned to whatever ubuntu provides. Please remove `/opt/arm/cmake/bin` from the path.

Reply:
Thanks Tristan, I've fixed that now.